README.md

```diff
@@ -63,6 +63,8 @@ Commands:
   script-login  Login to a snappy data server with a shell script using...
 ```
 
+Snappy stores the file in the data_server directory, in case you want to create a new
```

Comment:
Instead of talking about the change here, add a new section with heading 'Posting a File' after 'Help'. In that section copy the output of `snappy post-file --help`. Ping me before you resubmit the pr.